.github/workflows/code_freeze_end.yml

```diff
@@ -9,12 +9,15 @@ jobs:
   end_code_freeze:
     if: | 
       github.event_name == 'workflow_dispatch' 
-      || (github.event.milestone.title == 'Code Freeze' && github.event.milestone.state == 'closed') 
+      || (github.event.milestone.title == 'Code Freeze' && github.event.milestone.state == 'closed')
     runs-on: ubuntu-latest
     env:
       GITHUB_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
 
     steps:
+    - name: Clone dd-trace-dotnet repository
+      uses: actions/checkout@v3
```

Comment:
TBH, if we're going to clone the repo anyway, I would avoid the bash and just do it in Nuke